src/docsfy/postprocess.py

```
@@ -630,13 +638,13 @@ async def add_cross_links(
     finally:
         shutil.rmtree(pages_dir, ignore_errors=True)
 
-    if not success:
+    if result is None or not result.success:
         logger.warning(
             f"[{project_name}] add_cross_links: AI call failed, returning pages unchanged"
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)